GitFyle.Core.Api/Services/Foundations/Repositories/RepositoryService.Exceptions.cs

```diff
@@ -138,37 +138,37 @@ private async Task<RepositoryDependencyException> CreateAndLogCriticalDependency
         private async ValueTask<RepositoryDependencyValidationException>
             CreateAndLogDependencyValidationExceptionAsync(Xeption exception)
         {
-            var RepositoryDependencyValidationException = new RepositoryDependencyValidationException(
+            var repositoryDependencyValidationException = new RepositoryDependencyValidationException(
                 message: "Repository dependency validation error occurred, fix errors and try again.",
                 innerException: exception,
                 data: exception.Data);
 
-            await this.loggingBroker.LogErrorAsync(RepositoryDependencyValidationException);
+            await this.loggingBroker.LogErrorAsync(repositoryDependencyValidationException);
 
-            return RepositoryDependencyValidationException;
+            return repositoryDependencyValidationException;
         }
 
         private async ValueTask<RepositoryDependencyException> CreateAndLogDependencyExceptionAsync(Xeption exception)
         {
-            var RepositoryDependencyException = new RepositoryDependencyException(
+            var repositoryDependencyException = new RepositoryDependencyException(
                 message: "Repository dependency error occurred, contact support.",
                 innerException: exception);
 
-            await this.loggingBroker.LogErrorAsync(RepositoryDependencyException);
+            await this.loggingBroker.LogErrorAsync(repositoryDependencyException);
 
-            return RepositoryDependencyException;
+            return repositoryDependencyException;
         }
 
         private async ValueTask<RepositoryServiceException> CreateAndLogServiceExceptionAsync(
            Xeption exception)
         {
-            var RepositoryServiceException = new RepositoryServiceException(
+            var repositoryServiceException = new RepositoryServiceException(
                 message: "Service error occurred, contact support.",
```

Comment:
```suggestion
                message: "Repository service error occurred, contact support.",
```

I think the message string can also be fixed here.